UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.FulfilmentService/appsettings.local.overrides.json

```diff
@@ -55,7 +55,11 @@
     "BatchCommitDelayTimeInMilliseconds": 2000,
     "PosBatchCommitCutOffTimeInMinutes": 90,
     "PosBatchCommitDelayTimeInMilliseconds": 60000,
-    "TimeOutInMins": 3
+    "TimeOutInMins": 3,
+    "ContentInfo": "DVD INFO",
+    "Content": "Catalogue",
+    "CatalogueType": "ADC",
+    "AdcFolderPath": "ADC"
```

Comment:
Can we have single key like "Adc": "Adc" and use it as required as same value in both Key ?

**Note:** Need to change in class as well and wherever it get used.